README.md

```diff
@@ -46,23 +46,23 @@ The following chart shows the time it takes to run the 22 TPC-H queries against
 using a single executor with 8 cores. See the [Comet Benchmarking Guide](https://datafusion.apache.org/comet/contributor-guide/benchmarking.html)
 for details of the environment used for these benchmarks.
 
-When using Comet, the overall run time is reduced from 640 seconds to 331 seconds, very close to a 2x speedup.
+When using Comet, the overall run time is reduced from 626 seconds to 290 seconds, a 2.2x speedup.
```

Comment:
Note that these benchmarks used Spark 3.5.4 running in k8s, and previous benchmarks ran with Spark 3.4.3 in standalone mode without k8s/docker.

Reply:
Do we document the configuration used for the benchmark on k8s anywhere?

Reply:
We provide the full spark-submit scripts in the benchmarking guide, and there is a link to that page just before the results here.

Example: https://datafusion.apache.org/comet/contributor-guide/benchmark-results/tpc-h.html

This PR updates these scripts to use k8s.